pages/la-nov-20.tsx

```diff
@@ -0,0 +1,160 @@
+import { Fragment } from "react";
+import { NextPage } from "next";
+import Link from "next/link";
+
+import { data } from "../data";
+
+const LA: NextPage = () => {
+  return (
+    <>
+      <div className="mw-app center pb6-ns pb5 pt5 ph-app">
+        {/* Header */}
+        <div className="flex flex-column">
+          <Link href="/">
+            <a className="cool-red f6 fw7 lh-solid no-underline underline-pointer-hover pb4 mb3">
+              <svg
+                width="7"
+                height="10"
+                viewBox="0 0 7 10"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+                className="pr2"
+              >
+                <path d="M6 9L2 5L6 1" className="caret" strokeWidth="2" />
+              </svg>
+              All cities
+            </a>
+          </Link>
+
+          <div className="relative">
+            <img alt="DSA LA logo" src="/dsa-la@2x.png" className="h3 w-auto" />
```

Comment:
@tvler u cool with sizing logos like this? then they can all be sized the same way. yuri trained me to make things the same height 🙎‍♂️

Reply:
yea! good idea